snap/snapcraft.yaml

```diff
@@ -122,21 +135,47 @@ parts:
     - qt6-base-dev-tools
     stage-packages:
     - apparmor
-    - on amd64: [libgl1]
```

Comment:
How did determine that some packages were not needed?

Reply:
Well, the Snapcraft linter helped with some of it and then a bunch of trial and error.